helm/hpcc/templates/_helpers.tpl

```diff
@@ -101,36 +118,161 @@ Add ConfigMap volume for a component
 
 {{/*
 Add data volume mount
+If any storage planes are defined that name pvcs they will be mounted
 */}}
 {{- define "hpcc.addDataVolumeMount" -}}
+{{- /*Create local variables which always exist to avoid having to check if intermediate key values exist*/ -}}
+{{- $storage := (.root.Values.storage | default dict) -}}
+{{- $planes := ($storage.planes | default list) -}}
+{{- $dataStorage := ($storage.dataStorage | default dict) -}}
+{{- $daliStorage := ($storage.daliStorage | default dict) -}}
+{{- $dllStorage := ($storage.dllStorage | default dict) -}}
+{{- $daliStoragePlane := ($daliStorage.plane | default "") -}}
+{{- $dllStoragePlane := ($dllStorage.plane | default "") -}}
+{{- range $plane := $planes -}}
```

Comment:
@ghalliday - was there the idea that a Thor could have a list of accessible storage planes, by default all,
such that queries running on that Thor were only allowed/mounted those SP's, so that if a configuration wanted to, they could isolate SP's ?

Reply:
The current implementation is that thor/hthor/roxie has access to all storage planes (except dali/dlls).  Each engine has a default plane, but it can be overwritten by using the CLUSTER() attribute on an output.  (That side is currently a work in progress.)
We may want to restrict the planes somehow later, but that can be a future refinement.